.github/actions/test-network-setup/action.yaml

```diff
@@ -10,10 +10,9 @@ inputs:
   java-version:
     description: Version of JDK
     default: 11.x
-#  todo : set fabric target version ...
-#  fabric-version:
-#    description: Version of Fabric
-#    default: 2.5
+  fabric-version:
+    description: Version of Fabric
+    default: 2.4.7
```

Comment:
The intent is for latest fabric-samples (main branch) to always run against latest fabric, that is, don't specify a version. It seems whenever we specify a version we forget to increment it and it gets stale.

Reply:
I agree.  This area seems a little weak, and can be improved across samples as a whole.  We need to figure something out on this front if/when we start generating and publishing images as part of the samples build.

I un-pinned the CLI revs, all the tests should now run with the latest CLI.   Reminder to check in on this when the next minor rev of Fabric has been published.